extension.meta.xml

```diff
@@ -23,8 +23,11 @@
 		<!-- None -->
 	</dependencies>
 	<releases>
+		<release version="1.1.2" date="2016-07-21" min="2.3" max="2.x.x">
+			- Updated fixed issues #5 and #6, compatibility with php 7 
+		</release>
 		<release version="1.1.1" date="2016-03-07" min="2.3" max="2.x.x">
-			- Updated compatibility info
+			 Updated compatibility info
```

Comment:
Haha, still missing a - !

It's important since the content of the xml tag will be parsed by markdown!